Index: 1.JavaSyntax/src/com/javarush/task/task01/task0131/Solution.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- 1.JavaSyntax/src/com/javarush/task/task01/task0131/Solution.java	(date 1586061589428)
+++ 1.JavaSyntax/src/com/javarush/task/task01/task0131/Solution.java	(date 1586061589428)
@@ -0,0 +1,15 @@
+package com.javarush.task.task01.task0131;
+
+/* 
+Полнометражная картина
+*/
+
+public class Solution {
+    public static void main(String[] args) {
+        System.out.println(getMetreFromCentimetre(243));
+    }
+
+    public static int getMetreFromCentimetre(int centimetre) {
+        return centimetre / 100;//напишите тут ваш код
+    }
+}
\ No newline at end of file
